language.py

```diff
@@ -0,0 +1,35 @@
+import json
+import os
+import threading
+
+
+class Language:
+    LANGUAGE_CONFIG_FILE = 'languages.json'
+
+    def __init__(self, default_language):
+        self.default_language = default_language
+        self.__languages = {}
+        self.load_languages()
+
+    def load_languages(self):
+        if not os.path.exists(self.LANGUAGE_CONFIG_FILE):
+            return
+        lock = threading.Lock()
+        with lock:
+            with open(self.LANGUAGE_CONFIG_FILE) as f:
+                self.__languages = json.load(f)
+
+    def save_languages(self):
+        lock = threading.Lock()
+        with lock:
+            with open(self.LANGUAGE_CONFIG_FILE, 'w') as f:
+                json.dump(self.__languages, f, sort_keys=True, indent=2)
+
+    def add(self, guild, language):
+        self.__languages[str(guild.id)] = language
```

Comment:
in the others I created a dictionary with the guild's name also, to be able to give support. The guild id is fine for storage, but not human readable.

Reply:
Also, you copied my shitty `str(guild_id)` ... please remove the type cast here, it does not make sense.